RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -84,27 +78,30 @@ struct ManageSubscriptionsView: View {
                 background.edgesIgnoringSafeArea(.all)
             }
 
-            ScrollView {
-                VStack {
-                    if self.viewModel.isLoaded {
-                        SubtitleTextView(subtitle: self.viewModel.screen.subtitle)
+            if self.viewModel.isLoaded {
+                List {
 
-                        if let subscriptionInformation = self.viewModel.subscriptionInformation {
+                    if let subscriptionInformation = self.viewModel.subscriptionInformation {
+                        Section {
                             SubscriptionDetailsView(
                                 subscriptionInformation: subscriptionInformation,
                                 localization: self.localization,
                                 refundRequestStatusMessage: self.viewModel.refundRequestStatusMessage)
                         }
+                    }
 
+                    Section {
                         ManageSubscriptionsButtonsView(viewModel: self.viewModel,
                                                        loadingPath: self.$viewModel.loadingPath)
-                    } else {
-                        ProgressView()
-                            .progressViewStyle(CircularProgressViewStyle())
+                    } header: {
+                        if let subtitle = self.viewModel.screen.subtitle {
+                            Text(subtitle)
+                                .textCase(nil)
```

Comment:
by default, lists apply an uppercase style for section headers - this disables the uppercase style to make it use the exact subtitle text that's configured (which could be made uppercase by the developer)

Reply:
we would need to apply the text color here though as we were doing in `SubtitleTextView` right?

Reply:
resolving this since we talked about it separately 👍